.github/workflows/e2e.cluster.yaml

```diff
@@ -38,9 +38,6 @@ jobs:
       matrix:
         config-file:
           - cluster/zk/es/e2e.yaml
-          - cluster/zk/mysql/e2e.yaml
-          - cluster/zk/influxdb/e2e.yaml
-          - cluster/zk/iotdb/e2e.yaml
```

Comment:
Is it intentional? What's the rationale to delete these?

Reply:
All previous cluster tests are not working, as the coordinator is never set correctly. 
Now, based on discussions between Kai and me, for the cluster test cases, the key is to make sure the coordinator works, rather than the storage option. 
So, if we want to add more test cases, other coordinators should be tested in similar ways. But all existing cases are not relevant.
This is the reason it is removed here.